libnetwork/driver_plugin.py

```diff
@@ -12,27 +12,53 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from flask import Flask, jsonify, request
-import socket
 import logging
 import sys
 
 from pycalico.util import generate_cali_interface_name
 from subprocess32 import CalledProcessError
 from werkzeug.exceptions import HTTPException, default_exceptions
-from netaddr import IPNetwork
+from netaddr import IPAddress, IPNetwork
 from pycalico.datastore_datatypes import IF_PREFIX, Endpoint, IPPool
 from pycalico import netns
+from pycalico.util import get_hostname
 
 from datastore_libnetwork import LibnetworkDatastoreClient
 
+
+# Fixed MAC address used in the container
```

Comment:
You've added a load of comments, but IMHO they're pretty pointless. If you're going to add them I think you should take them a bit further. e.g. I can see that FIXED_MAC is a fixed mac address - can your comment say why we're using a fixed mac address and why we've chosen that value?